packages/preview/exzellenz-tum-thesis/0.1.0/template.typ

```diff
@@ -0,0 +1,87 @@
+#import "cover.typ": *
+#import "titlepage.typ": *
+#import "disclaimer.typ": *
+#import "acknowledgement.typ": *
+#import "abstract.typ": *
+
+
+#let exzellenz-tum-thesis(
+  degree: "The degree",
+  program: "The Program",
+  supervisor: "Your Supervisor",
+  advisors: ("The first advisor", "The second advisor"),
+  author: "The Author",
+  startDate: "The Startdate",
+  titleEn: "English Title",
+  titleDe: "German Title",
+  abstractEn: [English Abstract],
+  abstractDe: [German Abstract],
+  acknowledgements: [The acknowledgements],
+  submissionDate: "(Handover Date)",
+  showTitleInHeader: true,
+  draft: true,
+  body,
+) = {
+
+  let draft_string = ""
+  if draft{
+    draft_string = "DRAFT - "
+  }
+
+  set document(author: author, title: draft_string + titleEnglish)
```

Comment:
The template doesn't currently compile when checked out because `titleEnglish` should be `titleEn`. Not sure whether that is the only issue or not.

See [the end of this section](https://github.com/typst/packages?tab=readme-ov-file#templates) for how to test your template locally.